.gitignore

```diff
@@ -2,6 +2,7 @@
 .sbi_env/
 *sbi-logs/
 /docs/docs/tutorial/*
+/docs/docs/examples/*
```

Comment:
@janfb this is unrelated to this PR. The `png` and `.md` files of the `examples` had been tracked, which they should not (they should only live in the `gh-pages` branch. I removed them, thus also the diffs below.